CMakeLists.txt

```diff
@@ -340,6 +340,7 @@ install(FILES ${LIBDATACHANNEL_HEADERS}
 install(
   EXPORT datachannel-export
   NAMESPACE LibDatachannel::
+  FILE libdatachannel-config.cmake
```

Comment:
It should be `datachannel-config.cmake`, right? Rather than adding a `FILE` line, you could simply change `EXPORT datachannel-export` to `EXPORT datachannel-config`.

Reply:
> It should be `datachannel-config.cmake`, right?

No, it should be ``libdatachannel-config.cmake``, as we use:
```
find_package(libdatachannel CONFIG)
```
rather than
```
find_package(datachannel CONFIG)
```
to find the ``libdatachannel`` package.

> Rather than adding a `FILE` line, you could simply change the export name from `datachannel-export` to ` datachannel-config`.

Done.

Reply:
Do not merge, ``datachannel-config`` does not work, let me check again.

Reply:
Works now, the export name should indeed be ``libdatachannel-config`` rather than ``datachannel-config``.
Also see patch at [vcpkg](https://github.com/microsoft/vcpkg/blob/7b1016e10ef0434fe7045d320a9ee05b63e0efe9/ports/libdatachannel/fix-for-vcpkg.patch#L73).

Reply:
OK thanks :+1: